types/karma/index.d.ts

```diff
@@ -14,7 +14,10 @@ import Promise = require('bluebird');
 import https = require('https');
 import { Appender } from 'log4js';
 import { EventEmitter } from 'events';
+import constants = require('./lib/constants');
```

Comment:
you should probably `import * as constants from './lib/constants'`. it doesn't have a commonjs default export in the sources, so you shouldn't need to use `require`

Reply:
done, amended